main.py

```diff
@@ -47,27 +46,26 @@ def get_magazine_links(url):
 # --------- Gathering Links End ---------
 
 # --------- Parsing the Data --------- 
-# Creating data dictionary for each checked article
-# Note: For this project, I didn't add any controls so all links will be used
-def createDataDict(checkledLink):
-    global checkCounter
+# Parsing the data from each article link
+def ParseArticle(articleLink):
     global counter
     counter += 1
     
     # 429 means that the server is overloaded
```

Comment:
The last time I run the program the 429 error never occurred. So this section can be removed in future.